redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -260,6 +314,20 @@ internal open class UIViewLazyList :
     updateProcessor.onEndChanges()
     scrollProcessor.onEndChanges()
   }
+
+  override fun detach() {
+    // Reverse what we set up in init {}. This is strictly unnecessary but should maximize the
+    // number of Kotlin objects that are immediately eligible for garbage collection.
+    scrollProcessor.tableView = null
+    scrollProcessor.updateProcessor = null
+    updateProcessor.tableView = null
+    dataSource.updateProcessor = null
+    tableViewDelegate.tableView = null
+    tableViewDelegate.scrollProcessor = null
+    tableView.scrollProcessor = null
+    // tableView.dataSource = null // Don't clear the data source, it's weakly held anyway.
```

Comment:
nit: might as well delete the commented-out code here (the comment is useful though!)

Reply:
Fixed!